en/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md

```diff
@@ -0,0 +1,179 @@
+---
+id: applications-pfsense-fauxapi
+title: Pfsense Fauxapi
+---
+
+## Overview
+
+At its core FauxAPI simply reads the core pfSense config.xml file, converts it to JSON and returns to the API caller. 
+FauxAPI provides easy backup and restore API interfaces that by default store configuration backups on all configuration write operations thus it is very easy to roll-back
+even if the API user manages to deploy a “very broken” configuration.
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+
+* Applications
+* Firewall
+* Router
+* FauxAPI
```

Comment:
Should be the list of the Services templates no ?

Reply:
Yes. Done!